### PR TITLE
[RAP-244] feat: loadtest

### PR DESCRIPTION
**Pull request by UDtorrey**
## Background
Load testing of multiple services via automated methods was needed, resolution service had some tests defined for that effort.

## Changes
new definition of some tests in scala

## To Do
n/a

## Additional Deployment and/or Rollback Steps
n/a

## Code Review

This Pull Request was thoroughly reviewed and meets all Code Review Standards pertaining to:

- [ ] **Implementation** - satisfied acceptance criteria
- [ ] **Communication** - notified engineers/stakeholders about impactful changes
- [ ] **Error handling** - handled, logged & alerted on errors
- [x] **Documentation** - wrote readable code & commits, documented unintuitive code
- [ ] **Testing** - unit & E2E test coverage, tested in staging, DB migrations backwards compatible
- [x] **Security** - sanitized inputs, vetted dependencies, validated/secured API requests, no committed secrets
- [ ] **Performance** - optimized expensive algorithms & database queries

Please select all **applied** standards relevant to this PR.
## Confirmation

- [ ] **Assignee Confirmation** - I (the author) have filled out the template above
- [ ] **Reviewer Confirmation** - I (a/the reviewer) confirm 1) the author has filled out the template and checked the box that says **Assignee Confirmation** 2) I reviewed the code and selected all applicable items in the code review section.

### CODE REVIEW

File: loadtest/ResolutionAPISimulation.scala [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        .get("/status")
+        .headers(authHeaders)
+      // .check(
+      //   jmesPath("Blockchain.ETH.networkId")
+      //     .ofType[Int]
+      // )
+    )
+      .feed(domainNameFeeder)
+      .exec(
+        http("2 /domains/randomDomainName")
```
Comment (tunguyennnnn) [inline loadtest/ResolutionAPISimulation.scala:242]: we may want to add tests for `/domains` endpoint with different parameters (ex /domains?tlds=crypto` and well as bulk reverse endpoint. those seems to have performance penalty than these

Comment (tunguyennnnn) [reply loadtest/ResolutionAPISimulation.scala:242]: cc @sammyluo

Comment (UDtorrey) [reply loadtest/ResolutionAPISimulation.scala:313]: added tests for /domains?tlds=${randomTld}
as well as POST /reverse/query

File: loadtest/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+1. create new stream in [flood.io](https://app.flood.io/) (select your project, Streams, new stream; or update an existing stream)
+1. Set Advanced/Optional parameters using `-D` syntax similar to setting `JAVA_OPTS` when running locally
+1. in [flood.io](https://app.flood.io/) select the project & stream you created, go to launch tab, set load test parameters, and `Launch Test`
+
+each test execution in flood.io is called a 'flood'
+
+[example flood](https://app.flood.io/projects/99747/flood/2LNd1UszwdAJvHVP7zeeZUXp8fQ/grid/jOKjAif7R1iWMlhMA8081g/timeline/2023-02-06T20:32:30.000Z/2023-02-06T20:36:15.000Z?label=domainsExpandRecords4) in [flood.io](https://flood.io)
+
+## Demo
+see this series of [loom videos](https://loom.com/share/folder/2e5e6b2b9d0f42b09e857cca68ea8c00) for these tests in use with both flood.io and locally via gatling
```
Comment (tunguyennnnn) [inline loadtest/README.md:29]: 🤔 maybe we shouldn't have loom here? the repo and the loom link is public, I feel like we are opening for attack

**Review by tunguyennnnn — APPROVED** [review]

File: loadtest/README.md (hunk 2 above)
Comment (UDtorrey) [reply loadtest/README.md:29]: should we remove flood.io references as well?

Comment (tunguyennnnn) [reply loadtest/README.md:29]: the flood.io link for me is not accessible, but yea I think we should remove it too

**Review by tunguyennnnn — APPROVED** [review]

**Pull request merged**